### ISSUE TITLE
`cargo ws create` failed

### ISSUE THREAD

**Issue by holmofy** [pksunkara/cargo-workspaces#166]
[This](https://github.com/holmofy/autumn-boot) is my project. I failed when executing the command `cargo ws create examples/sea-orm`. The error message is very small and I can't fix the problem by myself:

```sh
~/rust/autumn-boot on master ❯ cargo ws create examples/sea-orm
✔ Name of the crate · sea-orm
✔ Type of the crate · binary
✔ Rust edition · 2021
    Creating binary (application) `sea-orm` package
note: see more `Cargo.toml` keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
error: unable to create crate
```

**Comment by pksunkara** [comment]
That's weird. What cargo-ws and rust version is this?

**Comment by holmofy** [comment]
~ ❯ rustc --version
rustc 1.79.0 (129f3b996 2024-06-10)
~ ❯ rustc --version
rustc 1.79.0 (129f3b996 2024-06-10)

**Comment by pksunkara** [comment]
Fixed in 0.3.2
